runtime/browser/xwalk_runner.cc

```diff
@@ -163,6 +169,48 @@ void XWalkRunner::OnRenderProcessWillLaunch(content::RenderProcessHost* host) {
       std::move(runtime_variables));
 }
 
+void XWalkRunner::InitializeEnvironmentVariablesForGoogleAPIs(
+    content::RenderProcessHost* host) {
+#if defined (OS_WIN)
+  application::Application* app =
+      app_system()->application_service()->
+      GetApplicationByRenderHostID(host->GetID());
+  if (!app)
+    return;
+  std::string app_name;
+  if (!app->data()->GetManifest()->GetString("xwalk_package_id", &app_name))
+    return;
+  std::string vendor =
+      app_name.substr(0, app_name.find_last_of("."));
```

Comment:
can we make a simpler registry key (any that is existing in manifest), so that we do not parse xwalk_package_id (not a major problem but still..)?